pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.56</version> <!-- or whatever the newest version available is -->
+    <version>4.1</version> <!-- or whatever the newest version available is -->
```

Comment:
This seems ill adviced:
https://github.com/jenkinsci/plugin-pom/blob/88cb710c1b7244bf5f885e16fc9f0baef1ef2661/pom.xml#L53
Please at least specify a `enkins.version` property

Reply:
Hi, @jetersen!
This change was to just update the parent pom version. It was also included in this PR as well, which is ready to be merged: https://github.com/jenkinsci/datadog-plugin/pull/57/files#diff-600376dffeb79835ede4a0b285078036R8 Adding a minimum jenkins version is out of the scope for this PR, but it’s a good idea that we should do in a later PR.